config-templates/config.php

```diff
@@ -861,6 +861,37 @@
      */
     'production' => true,
 
+    /*
+     * Set this option to adjust basic caching directives sent by module.php.
+     *
+     * Defaults to ['public' => true, 'max_age' => 86400]
+     */
+    // 'module.php.cache' => ['public' => true, 'max_age' => 86400],
+
+    /*
+     * Set this option to adjust expiration timestamp sent by module.php.
+     * The value (in seconds) is added to current time.
+     * If set to null, the Expiration header is not sent at all.
+     *
+     * Defaults to 10 * 60 (ten minutes).
+     */
+    // 'module.php.expires' => 10 * 60,
```

Comment:
I think this should go as well inside a general `caching` group (name again is orientative).